src/Global.h

```diff
@@ -22,6 +22,10 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 #include <string>
 
+#if (defined __APPLE__)
```

Comment:
Is this change related to the others?

Reply:
Yes :). The macros in global.h may not work correctly. The main change made in this PR ist a good example for that.